public/assets/css/main.css

```diff
@@ -21,12 +26,40 @@ p {
 	font-weight: 300;
 }
 
+a {
+	color: #FFF;
+}
+
 #logo {
-	margin: 5em auto 0 auto;
+	margin: auto;
 	width: 10em;
 	display: block;
 }
 
+section {
+	max-width: 42em;
+	margin: auto;
+	padding: 1em;
+
+	display: flex;
+	justify-content: space-between;
+	flex-direction: column;
+}
+
+figure {
+	margin: 0;
+	width: 100%;
```

Comment:
Why do you need this? I don't think it should be useful, and upon testing I don't see any change.